test/web-platform-tests/to-upstream/webmessaging/support/ChildWindowPostMessage.htm

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
This is a simplified version of https://github.com/web-platform-tests/wpt/blob/master/webmessaging/support/ChildWindowPostMessage.htm

The original file uses `event.source` which I changed to `parent` as a temp workaround.